src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -560,12 +559,12 @@ ${loaderJs}
 		});
 	}
 
-	createInset(cell: CodeCellViewModel, output: IProcessedOutput, cellTop: number, offset: number, shadowContent: string, preloads: Set<string>) {
+	async createInset(cell: CodeCellViewModel, output: IProcessedOutput, cellTop: number, offset: number, shadowContent: string, preloads: Set<string>) {
```

Comment:
I _think_ making this async is safe, since the display of content was already asynchronous behind the postMessage being sent.

Reply:
It's safe. Can we also make the call hierarchy explicit async then others won't use it incorrectly later on?